uint8 camera_action
---
bool result
